src/emulator/eventarcEmulator.ts

```diff
@@ -0,0 +1,169 @@
+import * as _ from "lodash";
+import * as express from "express";
```

Comment:
Nit - I prefer to add a newline between imports from external deps (like express) and imports from other files in this project